keyboards/handwired/ms_sculpt_mobile/keymaps/milestogo/keymap.c

```diff
@@ -1,20 +1,29 @@
-#include "babblePaste.h"
 #include QMK_KEYBOARD_H
 
-#define _QWR 0
-#define _CDH 2
-#define _SYM 3
-#define _MOV 4
-#define _TRAN 5
+#ifdef BOARD_GENERIC_STM32_F103
+#include "8x18_arm.h"
+#endif
```

Comment:
In theory, this shouldn't be needed, and should be handled by the `QMK_KEYBOARD_H` macro. 

If not, then it should be included in the `ms_sculpt_mobile.h` file, not here.